doc/sphinx/biblio.bib

```diff
@@ -136,7 +136,7 @@ @InProceedings{Del00
   pages       = {85--95},
   month       = {November},
   year        = {2000},
-  url         = {http://www.lirmm.fr/\%7Edelahaye/papers/ltac\%20(LPAR\%2700).pdf}
+  url         = {http://www.lirmm.fr/%7Edelahaye/papers/ltac%20(LPAR%2700).pdf}
```

Comment:
The problem when you omit the escaping is that the end of the line gets interpreted as a comment.

Reply:
Or maybe it's just GitHub which is stupid.